assets/src/components/profile/EmailSettings.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Button,
+  ContentCard,
+  Flex,
+  Switch,
+  useSetBreadcrumbs,
+} from '@pluralsh/design-system'
+import { LoginContext } from 'components/contexts'
+import { ScrollablePage } from 'components/utils/layout/ScrollablePage'
+import { Body2P } from 'components/utils/typography/Text'
+import { useUpdateUserMutation } from 'generated/graphql'
+import { useContext, useState } from 'react'
+import { PROFILE_BREADCRUMBS } from './MyProfile'
+
+export function EmailSettings() {
+  useSetBreadcrumbs(PROFILE_BREADCRUMBS)
+  const { me } = useContext(LoginContext)
+  const [subscribed, setSubscribed] = useState<boolean>(
+    !!me?.emailSettings?.digest
+  )
+  const [mutation, { loading }] = useUpdateUserMutation({
+    variables: {
+      attributes: { emailSettings: { digest: subscribed } },
+    },
+  })
+  const changed = subscribed !== !!me?.emailSettings?.digest
+
+  return (
+    <ScrollablePage heading="Profile">
+      <ContentCard
+        maxHeight="100%"
+        overflowY="auto"
+      >
+        <Flex
+          direction="column"
+          gap="large"
+        >
+          <Switch
+            checked={subscribed}
+            onChange={() => setSubscribed(!subscribed)}
+            css={{
+              flexDirection: 'row-reverse',
+              justifyContent: 'space-between',
+            }}
+          >
+            Subscribe to digest emails
```

Comment:
this should default to true fwiw (if undefined/null)

Reply:
updated